onnx/defs/tensor/defs.cc

```diff
@@ -1280,7 +1280,7 @@ When `reduction` is set to some reduction function `f`, `output` is calculated a
 output = np.copy(data)
 update_indices = indices.shape[:-1]
 for idx in np.ndindex(update_indices):
-    output[indices[idx]] = f(output[indices[idx]], updates[idx])
```

Comment:
It seems this code appears in a couple of other places. Is it possible to fix them as well?